syz-manager/html.go

```diff
@@ -55,6 +55,7 @@ func (mgr *Manager) initHTTP() {
 	handle("/filecover", mgr.httpFileCover)
 	handle("/input", mgr.httpInput)
 	handle("/debuginput", mgr.httpDebugInput)
+	handle("/canonicalmodules", mgr.canonicalModules)
```

Comment:
"canonical" is kind of an internal detail of how we deal with modules loaded at different addresses. It's not the only way, we can change it in future and this term does not make sense for end users.
So I think we need to drop "canonical" from user interfaces. E.g. this should be just "/modules", and the tool flag should be "-modules" instead of "-canon_mod".

Reply:
Thanks, done!